doc/main.html

```diff
@@ -2335,6 +2338,8 @@ <h3>Sample Workflow </h3>
         <a>https://doc.smpte-doc.org/ag-02/main/</a></li>
       <li><cite id="bib-smpte-ag-16">SMPTE AG 16</cite>, Standards Style Guide.
       <a>https://doc.smpte-doc.org/ag-16/main/</a></li>
+      <li><cite id="bib-ag-29">SMPTE AG-29</cite>, Document Library: URL Spec (Persistent URIs)
```

Comment:
```suggestion
      <li><cite id="bib-ag-29">SMPTE AG-29</cite>, Persistent URIs for identifying SMPTE documents
```

Reply:
This is the title I added already to this repo, see https://github.com/SMPTE/document-url-spec